src/Maestro/Maestro.Web/Models/SubscriptionDependencyDetails.cs

```diff
@@ -0,0 +1,50 @@
+using System.Collections.Generic;
+using System.Linq;
+using Maestro.Data;
+using Maestro.Data.Models;
+using Microsoft.DotNet.DarcLib;
+using Microsoft.DotNet.DarcLib.HealthMetrics;
+
+namespace Maestro.Web
+{
+    public class SubscriptionDependencyDetails
+    {
+        public readonly string Repository;
+        public readonly string Branch;
+
+        public List<Subscription> Subscriptions { get; private set; }
```

Comment:
None of these should have setters.  This type is actually immutable (only the constructor sets any values), so you don't need the setters (constructors are allowed to set "get only auto-properties" because magic)